packages/ui-table/src/Table/Body/index.tsx

```diff
@@ -70,11 +71,17 @@ class Body extends Component<TableBodyProps> {
         css={styles?.body}
         role={isStacked ? 'rowgroup' : undefined}
       >
-        {Children.map(children as RowChild[], (child) =>
+        {Children.map(children as any[], (child) =>
           safeCloneElement(child, {
             key: child.props.name,
+            // Sent down for compatibility with custom components
+            // TODO DEPRECATED, remove in next version
```

Comment:
I may add a specific version number (v11 or v12) here otherwise we could forget this and it may stay here forever.

Reply:
fixed